app/assets/stylesheets/application.css.scss

```diff
@@ -16,76 +16,77 @@
 
  */
 @import 'foundation-icons';
+
+/* CUSTOM VARIABLES */
+
 $boost-red: #B72039;
```

Comment:
Could be worth making a _variables.scss or _colours.scss file to store colours and then import in that file above

Reply:
Done in 82a524a